cmd/core-command/res/configuration.toml

```diff
@@ -51,7 +51,11 @@ Protocol = 'https'
 RootCaCertPath = '/vault/config/pki/EdgeXFoundryCA/EdgeXFoundryCA.pem'
 ServerName = 'localhost'
 TokenFile = '/vault/config/assets/resp-init.json'
-  [SecretStore.Authentication]
+# Number of attemtps to retry retrieving secrets before failing to start the service.
+AdditionalRetryAttempts = 10
+# Amount of time to wait before attempting another retry in nanoseconds.
+RetryWaitPeriod =  1000000000
+[SecretStore.Authentication]
```

Comment:
Is indentation important here?  Before, chapter heading was indented 2 spaces.  Now it isn't.  Commenting once; occurs multiple times.

Reply:
FWIW TOML format doesn't care, but we should try to keep indentation consistent for readability